src/lib/remote/markdown.remote.ts

```diff
@@ -0,0 +1,15 @@
+import { query } from '$app/server';
+import { readFile } from 'fs/promises';
+import { join } from 'path';
+
+const getMarkdownContent = async (routeId: string | null) => {
+    if (!routeId) return null;
+    try {
+        const basePath = join(process.cwd(), 'src', 'routes', routeId, '+page.markdoc');
+        return await readFile(basePath, 'utf-8');
```

Comment:
can we make sure they can only read files in the `routes` folder?

otherwise its a security issue by passing things like `../../../`